src/common/config/defaultPreferences.ts

```diff
@@ -12,12 +12,14 @@ import os from 'os';
 
 import {ConfigV3} from 'types/config';
 
-export const getDefaultDownloadLocation = (): string => {
+export const getDefaultDownloadLocation = (): string | undefined => {
     // eslint-disable-next-line no-undef
```

Comment:
Are all these comments still relevant? This one in particular doesn't look needed